### ISSUE TITLE
Progress components 

### ISSUE THREAD

**Issue by jmdrawneek** [First10/royal-canin-design-language#45]
## Progress components 
Markup needs refactoring to use actual numbers, not CSS generated numbers. This is a breach of WCAG https://www.w3.org/TR/WCAG20-TECHS/F87.html
Numeric counters should be content so screen readers work correctly.

**Comment by stephen9001** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Progress components](https://trello.com/c/oDG9mvZk/193-progress-components)
